Sources/Shared/Models/VIMConnection.m

```diff
@@ -67,6 +67,7 @@
 NSString *const VIMConnectionNameBlockUser = @"block";
 NSString *const VIMConnectionNameLiveStats = @"live_stats";
 NSString *const VIMConnectionNameUploadAttempt = @"upload_attempt";
+NSString *const VIMConnectionNamePublishToSocialJob = @"publish_to_social_job";
```

Comment:
Seems  should be `publish_to_social` (i.e. without "job" suffix). I checked via https://pts-staging.ci.vimeows.com/, there no `publish_to_social_job` for video objects, only `publish_to_social`

blueprint https://github.vimeows.com/Vimeo-API/design/pull/148/files#diff-4dafa2f9b34cb46c64cb8ef8d9694149R124